etc/greenboot/green.d/00_grub_fallback.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+# We need to determine whether this boot was a fallback boot
+if grub2-editenv list | grep -q "fallback_active=true"; then
```

Comment:
Looks like this should be `boot_counter=-1` now?
Also let's wrap regex with `^` and `$` to be safe.